OpenIGTLinkIF/MRML/vtkSlicerOpenIGTLinkCommand.h

```diff
@@ -73,6 +73,8 @@ class VTK_SLICER_OPENIGTLINKIF_MODULE_MRML_EXPORT vtkSlicerOpenIGTLinkCommand :
   std::string GetResponseMetaDataElement(std::string key);
 
   std::map<std::string, std::pair<IANA_ENCODING_TYPE, std::string> > GetCommandMetaData();
+	std::map<std::string, std::pair<IANA_ENCODING_TYPE, std::string> > GetResponseMetaData();
```

Comment:
spaces instead of tab

Reply:
oops, editor was misconfigured I guess - good catch